Test/output_test.py

```diff
@@ -488,12 +490,18 @@
 >>> cm_file.weights
 >>> cm_file.transpose
 False
+>>> cm_file.imbalance
+False
 >>> cm_file.matrix == {'1': {'1': 1, '2': 1, '0': 0}, '2': {'1': 2, '2': 3, '0': 0}, '0': {'1': 0, '2': 2, '0': 3}}
 True
 >>> cm_file.actual_vector == ['1', '1', '2', '2', '2', '2', '2', '0', '0', '0', '0', '0']
 True
 >>> cm_file.predict_vector == ['1', '2', '1', '1', '2', '2', '2', '2', '2', '0', '0', '0']
 True
+>>> json.dump({"Actual-Vector": ['1', '1', '2', '2', '2', '2', '2', '0', '0', '0', '0', '0'], "Digit": 5, "Predict-Vector": ['1', '2', '1', '1', '2', '2', '2', '2', '2', '0', '0', '0'], "Matrix": {"0": {"0": 3, "1": 0, "2": 2}, "1": {"0": 0, "1": 1, "2": 1}, "2": {"0": 0, "1": 2, "2": 3}}, "Is_imbalanced": True},open("test8.obj","w"))
```

Comment:
In this case, you should save a JSON file without `Is_imbalanced` flag (previous versions) then load it and check backward compatibility.

Reply:
Last test (line 488 and 493) isn't that you want?
Here I just wanted to see the changes if we load the confusion matrix from obj file.